app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/repository/di/AuthenticatorBridgeModule.kt

```diff
@@ -33,10 +38,21 @@ object AuthenticatorBridgeModule {
     fun provideAuthenticatorBridgeManager(
         factory: AuthenticatorBridgeFactory,
         symmetricKeyStorageProvider: SymmetricKeyStorageProvider,
-    ): AuthenticatorBridgeManager = factory.getAuthenticatorBridgeManager(
-        connectionType = AuthenticatorBridgeConnectionType.DEV,
-        symmetricKeyStorageProvider = symmetricKeyStorageProvider,
-    )
+        featureFlagManager: FeatureFlagManager,
+    ): AuthenticatorBridgeManager =
+        if (featureFlagManager.getFeatureFlag(LocalFeatureFlag.PasswordManagerSync)) {
+            factory.getAuthenticatorBridgeManager(
+                connectionType = AuthenticatorBridgeConnectionType.DEV,
```

Comment:
We're going to go in later and how this type based someone on the current build type, right?

Reply:
Correct. I was actually gonna do that as a small follow up here shortly.

The description is a bit off, but here's the the ticket: https://livefront.atlassian.net/browse/BITAU-68